crates/pep508-rs/src/unnamed.rs

```diff
@@ -84,6 +67,17 @@ impl Serialize for UnnamedRequirement {
     }
 }
 
+impl UnnamedRequirement {
+    /// Returns whether the markers apply for the given environment
+    pub fn evaluate_markers(&self, env: &MarkerEnvironment, extras: &[ExtraName]) -> bool {
+        if let Some(marker) = &self.marker {
+            marker.evaluate(env, extras)
+        } else {
+            true
+        }
+    }
+}
+
```

Comment:
Seems like the changes in this file could be done in a separate commit, even merged directly to `main`.

Reply:
Rebase fallout from https://github.com/astral-sh/uv/pull/3186, i'll revert the changes in this file